### ISSUE TITLE
Investigate YouTube Videos not playing in NZ

### ISSUE THREAD

**Issue by georgeblahblah** [guardian/dotcom-rendering#9478]
UserHelp have forwarded reports that videos are not playable for some readers in New Zealand. We should investigate to see what might cause this.

Please update [this thread](https://groups.google.com/a/guardian.co.uk/g/dotcom.platform/c/3oc5FOEhnZ0) with any important information.

**Comment by arelra** [comment]
I noticed when using a VPN to connect to NZ that an error is thrown from `consent-management-platform`:

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'consents')
    at index.esm.js:145:27
    at p (regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (regeneratorRuntime.js:125:22)
    at Generator.next (regeneratorRuntime.js:69:21)
    at r (asyncToGenerator.js:3:20)
    at c (asyncToGenerator.js:22:9)
```

Which I believe is referring to this line:
https://github.com/guardian/consent-management-platform/blob/0f46ea184e06a89886b175cf1bd64962770c9875/src/tcfv2/getConsentState.ts#L33

I will raise with T&C

**Comment by arelra** [comment]
T&C have two potential causes:

1) An error with Sourcepoint:

From Sourcepoint:

```
We had an issue with the gdprApplies flag which was fixed on Tuesday 14th.

The issue was with gdprApplies=true for all countries other than US, EEA and Canada where it was showing as false instead of true. 
 
The banner was still showing, it's just that the consent wasn't being read by the vendors. 
```

2. An error with the CMP

From T&C:

```
we had a version of CMP that was not producing the Types properly after a TS Config alteration.  

James Gorrie spotted and then pinned the version of CMP to the previous version to remove that issue.  I can't find where/when James notified us but it was about 2 weeks ago - the day of/after the release for 13.7.2 and the fixed version was released last week with the release of 13.7.3.
```

We have not determined a root cause of this issue and we were unable to replicate again so this can be closed.
